dependencies.md

```diff
@@ -392,3 +392,8 @@
 
 -   **version**: ^6.2.1
 -   **description**: offload cpu intensive work
+
+##### [better-queue](https://www.npmjs.com/package/better-queue)
```

Comment:
why are we using a different dependency for this queue ? can we use the same dependency for both commands queue and transactions queue ?